retis/src/core/probe/manager.rs

```diff
@@ -112,11 +111,7 @@ impl ProbeManager {
             builder
                 .maps
                 .iter()
-                .try_for_each(|(name, fd)| p.reuse_map(name, *fd))?;
-            builder
-                .global_probes_options
-                .iter()
-                .try_for_each(|o| p.set_option(o.clone()))
+                .try_for_each(|(name, fd)| p.reuse_map(name, *fd))
```

Comment:
Commenting here as it's related: we're also missing a `reuse_map` call for probes inserted at runtime, otherwise if they contain specific hooks (which is not the case ATM) those won't work as expected. The fix is the same (calling `reuse_map` before loading the probe) and then I guess the above can be removed too.